Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -53,25 +53,24 @@ public ItSystemUsage()
             AssociatedDataProcessingRegistrations = new List<DataProcessingRegistration>();
         }
 
-        public bool IsActive
+        public bool IsActiveAccordingToDateFields
```

Comment:
rembmer to rename references to the old property

probably not the only place: 
![image](https://user-images.githubusercontent.com/13116889/190112950-c873c745-baa6-4544-b56f-e7e75df0453c.png)